integrationTests/singleShard/block/interceptedRequestHdr_test.go

```diff
@@ -33,7 +33,7 @@ func TestNode_GenerateSendInterceptHeaderByNonceWithNetMessenger(t *testing.T) {
 
 	shardCoordinator := &sharding.OneShardCoordinator{}
 
-	fmt.Println("Requestor:")
+	fmt.Println("Requester:")
 	nRequestor, mesRequestor, _, resolversFinder := createNetNode(
 		dPoolRequestor,
 		storeRequestor,
```

Comment:
requestor?

Reply:
Removed all occurrences from the project